### ISSUE TITLE
Is hardware acceleration supported for static files?

### ISSUE THREAD

**Issue by strig** [m1k1o/go-transcode#40]
I'm using `--device=/dev/dri:/dev/dri` in my compose file but it doesn't seem to make any difference in CPU usage vs leaving it out. Am I missing some other config? Thanks

**Comment by m1k1o** [comment]
No, currently not implemented. Only for live transcoding. But should not be a big change to add it there.

**Comment by strig** [comment]
Gotcha, thanks. Might take a crack at it myself but don't know much about go or ffmpeg

**Comment by metal450** [comment]
Would also love to see this, for hardware transcoding of a static library hosted on Synology (https://timothybasanov.com/2018/12/08/hardware-accelerated-h264-encoding-synology-nas.html)
